lib/DxilValidation/DxilValidation.cpp

```diff
@@ -3921,6 +3921,18 @@ static void ValidateGlobalVariables(ValidationContext &ValCtx) {
     Rule = ValidationRule::SmMaxMSSMSize;
     MaxSize = DXIL::kMaxMSSMSize;
   }
+
+  // Check if the entry function has attribute to override TGSM size.
+  if (M.HasDxilEntryProps(M.GetEntryFunction())) {
+    DxilEntryProps &EntryProps = M.GetDxilEntryProps(M.GetEntryFunction());
+    if (EntryProps.props.IsCS()) {
+      unsigned SpecifiedTGSMSize = EntryProps.props.groupSharedLimitBytes;
+      if (SpecifiedTGSMSize > 0) {
+      MaxSize = SpecifiedTGSMSize;
+      }
```

Comment:
```suggestion
      if (SpecifiedTGSMSize > 0)
        MaxSize = SpecifiedTGSMSize;
```

LLVM coding standards say to omit braces here.

Something's also up with the formatting.  Did the format-checker spot it?  Anyway, clang-format should fix this for you.